src/components/verification-grid/verification-grid.ts

```diff
@@ -1123,11 +1264,68 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
     `;
   }
 
+  // TODO: this function could definitely be refactored
```

Comment:
agreed. More of this should be in another place.

Ideally: `tile.subject.skipUndecided(requiredDecisions)`